cmd/geth/accountcmd_test.go

```diff
@@ -88,6 +88,50 @@ Path of the secret key file: .*UTC--.+--[0-9a-f]{40}
 `)
 }
 
+func hexadecimal(count int) string {
+	chars := "0123456789abcdef"
+	var sb strings.Builder
+	for i := 0; i < count; i++ {
+		c := string(chars[i%len(chars)])
+		sb.WriteString(c)
+	}
+	return sb.String()
+}
```

Comment:
There's really no point in this method. Just add a constant to wherever you need it. You are replacing two string literals with a string builder that just makes the same thing anyway.

Reply:
Removed